lpc845-hal/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name    = "lpc845-hal"
+version = "0.4.0"
+authors = [
+    "David Sawatzke <david@sawatzke.de>",
+    "Hanno Braun <hanno@braun-robotics.com>"
+]
+edition = "2018"
+
+description   = "Hardware Abstraction Layer (HAL) for the NXP LPC82x series of ARM Cortex-M0+ microcontroller"
+documentation = "https://docs.rs/lpc845-hal"
+repository    = "https://github.com/lpc-rs/lpc82x-hal"
+license       = "0BSD"
+readme        = "README.md"
+categories    = ["embedded", "hardware-support", "no-std"]
+keywords      = ["arm", "cortex-m", "nxp", "lpc", "embedded-hal-impl"]
+
+
+[badges]
+maintenance = { status = "actively-developed" }
```

Comment:
I've been using these badges for a while, but changed my opinion since and started removing them from my projects. Two reasons for this:
1. I think they're too coarse-grained. Is a project that I update once a year actively developed? Not really, but it doesn't fit any of the other categories either.
2. If a project becomes unmaintained, I think it's likely that those badges become just as unmaintained as the project (i.e. the former maintainer forgets to update them).

I don't feel strongly about this, so feel free to keep it, if you want to. I'd prefer to remove it though.